packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -601,9 +602,9 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
   ): React.ReactNode {
     const { contextualMenuItemAs: ChildrenRenderer = ContextualMenuItem } = this.props;
     const { itemProps } = item;
-
+    const divHtmlProperties = itemProps && getNativeProps(itemProps, divProperties);
     return (
-      <div className={this._classNames.header} style={item.style}>
+      <div className={this._classNames.header} style={item.style} {...divHtmlProperties}>
```

Comment:
```suggestion
      <div className={this._classNames.header} {...divHtmlProperties} style={item.style}>
```

Reply:
We probably should move it over one more, I don't think that class name should get overridden.